### ISSUE TITLE
failing tests

### ISSUE THREAD

**Issue by petrelharp** [popsim-consortium/stdpopsim#1523]
I'm getting these errors:
```
FAILED tests/test_models.py::TestOrySatBottleneckMigration_3C07::test_mutation_rate_match - assert 9.033333333333333e-09 == 3.2e-09
FAILED tests/test_models.py::TestMutationRates::test_params_match_docs_tables - assert 9.033333333333333e-09 == 9.03e-09
```

One of these is precision. This should be easy to fix.

The other is a legit error, in `OrySat`. What's up with this?

**Comment by nspope** [comment]
The discussion regarding mutation rate for that OrySat model (vs species mutation rate) is here: https://github.com/popsim-consortium/stdpopsim/issues/1459#issuecomment-1546509693

I'll take a look and try to figure out where the discrepancy came in. Not sure why CI didn't catch it before.

**Comment by petrelharp** [comment]
Okay I see what happened here:
1. OrySat was added in #1453 , including this demographic model
2. The model was QC'ed in #1459 in April, but the PR wasn't merged
3. Some of the parameters were *changed* in #1482, in June
4. I noticed today that #1459 with the QC wasn't merged but had passed the tests, so I merged

However, those tests in (2) had passed before the changes in (3).

Note that the only substantive change in #1482 was to the mutation rate; the others are (good) semantic changes.
